tests/test_client_virtual_account.py

```diff
@@ -0,0 +1,89 @@
+import responses
+import json
+
+from .helpers import mock_file, ClientTestCase
+
+
+class TestClientVirtualAccount(ClientTestCase):
+
+    def setUp(self):
+        super(TestClientVirtualAccount, self).setUp()
+        self.base_url = '{}/virtual_accounts'.format(self.base_url)
+
+    @responses.activate
+    def test_virtual_accounts_all(self):
+        result = mock_file('virtual_accounts_collection')
+        url = self.base_url
+        responses.add(responses.GET,
+                      url,
+                      status=200,
+                      body=json.dumps(result),
+                      match_querystring=True)
+
+        self.assertEqual(self.client.virtual_account.all(), result)
+
+    @responses.activate
+    def test_virtual_accounts_all_with_options(self):
+        count = 1
+        result = mock_file('virtual_accounts_collection_with_one_item')
+        url = '{}?count={}'.format(self.base_url, count)
+        responses.add(responses.GET,
+                      url,
+                      status=200,
+                      body=json.dumps(result),
+                      match_querystring=True)
+
+        self.assertEqual(self.client.virtual_account.all(
```

Comment:
Needs more asserts on the data itself?

Reply:
@captn3m0 it asserts if both dict are equal not only count

Reply:
I'll add the additional assertions